core/lib/dal/sqlx-data.json

```diff
@@ -0,0 +1,3 @@
+{
+  "db": "PostgreSQL"
+}
```

Comment:
hmmm surprised to see this here. Do you use an old sqlx version?

Reply:
Probably, will check it.